Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFamilyInstance.cs

```diff
@@ -172,34 +172,50 @@ public ApplicationObject FamilyInstanceToNative(BuiltElements.Revit.FamilyInstan
         else
           familyInstance = Doc.Create.NewFamilyInstance(basePoint, familySymbol, level, StructuralType.NonStructural);
       }
-
-      //required for face flipping to work!
-      Doc.Regenerate();
-
-      if (familyInstance.CanFlipHand && speckleFi.handFlipped != familyInstance.HandFlipped)
-        familyInstance.flipHand();
-
-      if (familyInstance.CanFlipFacing && speckleFi.facingFlipped != familyInstance.FacingFlipped)
-        familyInstance.flipFacing();
-
-      // NOTE: do not check for the CanRotate prop as it doesn't work (at least on some families I tried)!
-      // some point based families don't have a rotation, so keep this in a try catch
-      try
-      {
-        if (speckleFi.rotation != (familyInstance.Location as LocationPoint).Rotation)
-        {
-          var axis = DB.Line.CreateBound(new XYZ(basePoint.X, basePoint.Y, 0), new XYZ(basePoint.X, basePoint.Y, 1000));
-          (familyInstance.Location as LocationPoint).Rotate(axis, speckleFi.rotation - (familyInstance.Location as LocationPoint).Rotation);
-        }
-      }
-      catch { }
-
+      #region OLD ROTATION CODE
```

Comment:
Why is this region commented out in the FamilyInstance class? Let's not make any changes to this class and fix any rotation problems in a separate issue 0 please remove these changes :)

Reply:
go it, you can discard my changes